<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainThree</class>
 <widget class="QMainWindow" name="MainThree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>394</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Facts</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/icoo.png</normaloff>../../../Downloads/icoo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {

     border-radius: 10px;
	background-color: lightblue;;
    padding: 2px 4px;
   border: 2px groove DarkCyan;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {

	background-color: rgb(240, 254, 255);
     border-radius: 10px;
    padding: 2px 4px;

   border: 2px groove DarkCyan;
}
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     border-radius: 10px;
    padding: 2px 4px;
   	
   border: 2px groove DarkCyan;
}</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
    /* border-radius: 10px;
    padding: 2px 4px;
   	background-color: ;*/

color:white;
   /*border: 1px groove DarkCyan;*/
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string>Enter amount of plastic bottles (1-9) :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.159091 rgba(95, 255, 120, 248), stop:0.363636 rgba(63, 85, 255, 237), stop:0.573864 rgba(95, 127, 255, 237), stop:0.784091 rgba(95, 248, 255, 244));
     border-radius: 10px;
    padding: 2px 4px;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
color: white;
}</string>
    </property>
    <property name="text">
     <string>What Plastic Can Become?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>211</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     /*border-radius: 10px;*/
    padding: 2px 4px;
   border: 2px groove DarkCyan;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/trainer2_adobespark.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>181</width>
      <height>171</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.159091 rgba(95, 255, 120, 248), stop:0.363636 rgba(63, 85, 255, 237), stop:0.573864 rgba(95, 127, 255, 237), stop:0.784091 rgba(95, 248, 255, 244));
     border-radius: 10px;
    padding: 2px 4px;
   border: 2px groove DarkCyan;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>101</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/b2p__adobespark.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>541</width>
      <height>361</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../Downloads/fon_adobespark.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0.159091 rgba(95, 255, 120, 248), stop:0.363636 rgba(63, 85, 255, 237), stop:0.573864 rgba(95, 127, 255, 237), stop:0.784091 rgba(95, 248, 255, 244));
     border-radius: 10px;
    padding: 2px 4px;
   border: 2px groove DarkCyan;
}
QPushButton:pressed {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
    </property>
    <property name="text">
     <string>Go Back</string>
    </property>
   </widget>
   <zorder>label_7</zorder>
   <zorder>label_6</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>538</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
